jd-geek-source/src/app/app.component.scss

```diff
@@ -0,0 +1,92 @@
+@import '../constants/gutters.scss';
+.main-gutter {
+  column-gap: $MAIN_GUTTER_WIDTH;
+}
+
+.half-gutter {
+  column-gap: $MAIN_GUTTER_WIDTH/2;
+}
+
+.double-gutter {
+  column-gap: $MAIN_GUTTER_WIDTH*2;
+}
+
+.toolbar {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  background-color: #123e69;
+  color: white;
+  font-weight: 600;
+}
+
+.toolbar img {
+  margin: 0 16px;
+}
+
+.running-logo {
+  position:relative;
+  height: 20px;
+  width: 160px;
+  margin: 0 16px;
+  text-align: right;
+}
+
+.content {
+  display: flex;
+  margin: 82px auto 32px;
+  padding: 0 16px;
+  max-width: 1180px;
+  flex-direction: column;
+  align-items: center;
+}
+
+.spacer {
+  flex: 1;
+}
+
+footer {
+  margin-top: 70%;
+  display: flex;
+  align-items: center;
+  line-height: 20px;
+}
+
+footer a {
+  display: flex;
+  align-items: center;
+}
+
+  /* Responsive Styles */
+  @media screen and (max-width: 767px) {
```

Comment:
media queries are a great use case for mixins

Reply:
Those are from the angular default project creation.  Just haven't removed them yet.